api/config/cdr_config_local.json

```diff
@@ -104,7 +99,6 @@
       "cdrVersionId": 6,
       "isDefault": false,
       "name": "Synthetic Dataset v3 with Microarray and WGS",
-      "dataAccessLevel": 1,
```

Comment:
There's a lot of surface area here - I'm not yet convinced that it's safe to remove this at the same time as which you're removing code that still references this value, e.g `findAuthorizedCdrVersions()`. Please convince me, or else stage the code change separate from the CdrConfig value change. e.g. - what is the value on the server right after I push this config, but before I get the new API server code?

Reply:
good point.  I'm not sure I understand the sequence of deployment steps well enough to be confident here.  I'll revert.